spec/app/models/mdm/web_page_spec.rb

```diff
@@ -1,8 +1,61 @@
+require 'webrick'
 RSpec.describe Mdm::WebPage, type: :model do
   it_should_behave_like 'Metasploit::Concern.run'
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
+      let(:string_cookie) do
+        "test_name=test_value"
+      end
+
+      let(:hash_cookie) do
+        {
+          name: 'test name',
+          value: 'test value'
+        }
+      end
+
+      let(:webrick_cookie) do
```

Comment:
This `let` is only used in `context 'with WEBrick::Cookie'`, so it should move into just that context where you can rename it to `let(:cookie)`